docs/src/siteConfig.js

```diff
@@ -5,9 +5,10 @@ export const siteConfig = {
   npmUrl: 'https://www.npmjs.com/package/@hyperjumptech/monika',
   repoUrl: 'https://github.com/hyperjumptech/monika',
   algolia: {
-    appId: '',
-    apiKey: '',
-    indexName: '', // algoliaOptions: {
+    appId: '5O2YZGK4NG',
+    apiKey: '45f684bf6230e7980f10b4a9209fce83',
+    indexName: 'Foods',
```

Comment:
use different name for the index, something related unlike `Foods`.

Reply:
ok, renamed to monika-dokumentation